.github/workflows/build.yml

```diff
@@ -228,9 +252,20 @@ jobs:
               }
             }
 
+      - name: Notify Slack channel on job failure
+        if: failure() && github.event_name == 'push'
```

Comment:
does the `github.event_name == 'push'` what is ensuring it's on main?

Reply:
The workflow is only triggered on pushes to main: https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/.github/workflows/build.yml#L4-L6
So we only need a condition on push vs pull_request